src/mail-app/settings/MailExportViewer.ts

```diff
@@ -0,0 +1,43 @@
+import m, { Children } from "mithril"
+import { assertMainOrNode, isDesktop } from "../../common/api/common/Env"
+import { lang } from "../../common/misc/LanguageViewModel"
+import { UpdatableSettingsViewer } from "../../common/settings/Interfaces.js"
+import { mailLocator } from "../mailLocator.js"
+import { MailExportSettings } from "./MailExportSettings"
+import { MailExportController } from "../native/main/MailExportController.js"
+
+assertMainOrNode()
+
+export class MailExportViewer implements UpdatableSettingsViewer {
+	private mailExportController: MailExportController | null = null
+
+	constructor() {
+		if (isDesktop()) {
+			// export is only available on desktop
+			mailLocator.mailExportController().then((controller) => {
+				this.mailExportController = controller
+				m.redraw()
```

Comment:
since this is not a "real" component you need to `this.view = this.view.bind(this)` or fun things will happen